clients/camunda-spring-boot-starter/pom.xml

```diff
@@ -163,6 +163,11 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-aspects</artifactId>
```

Comment:
❓ Is this used ?

Reply:
I think that's what the checks are complaining about, I seemed to have picked the wrong dep for the aspectJ classes needed in the test will check

Reply:
nope